smithy-swift-codegen-test-utils/src/main/kotlin/software/amazon/smithy/swift/codegen/test/utils/TestProtocolGenerator.kt

```diff
@@ -5,67 +5,37 @@
 
 package software.amazon.smithy.swift.codegen.test.utils
 
-import software.amazon.smithy.codegen.core.Symbol
```

Comment:
Duplicate code and encoder/decoder references eliminated from this test protocol generator.

In particular, custom error generation code is not needed and is deleted, the standard error generation may be used.